### PR TITLE
[azure-metrics-exporter] Add path azure metrics exporter

### LINKED ISSUE #15: [Azure-Metrics-Exporter] Metrics Path from Probes Config not handled in Service Monitor endpoint path
Hi,

Azure-Metrics-Exporter metrics path config e.g /probe/metrics/list is not handled in the service. monitor created by the helm chart.

### PR DESCRIPTION
**Pull request by Wihrt**
#### What this PR does / why we need it

Currently, it is impossible to change the path for metrics probes.
It is using the default path `/metrics` that point on the azure-metrics-exporter pod and not on the metrics fetched from Azure.
The default path is now `/probe/metrics/list` and it is possible to override it, if needed.

#### Which issue this PR fixes

- fixes #15

#### Checklist

- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[azure-metrics-exporter]`)

### CODE REVIEW

**Comment by SimonDreher** [comment]
@mblaschke Could you please take a look at this PR? We're running into the exact same problem and this would perfectly fix it. Thanks :)

**Comment by mblaschke** [comment]
i would prefer `probeDefaultPath` instead of `path` to avoid the confusion that this could be the path of the ServiceMonitor for `/metrics` 

any thoughts? or maybe move all the default configuration for probes into eg. `probeDefault`?

**Comment by SimonDreher** [comment]
I would prefer to name it `path` to be similar to the attribute in ServiceMonitor. I always find it confusing if charts invent their own names where I then need to keep a mapping in my head. (But just my personal preference)

I like the idea of `probeDefaults` :+1:

File: charts/azure-metrics-exporter/Chart.yaml [hunk 1]
```diff
@@ -3,13 +3,13 @@ name: azure-metrics-exporter
 type: application
 description: A Helm chart for azure-metrics-exporter
 home: https://github.com/webdevops/azure-metrics-exporter
-version: 1.0.4
+version: 1.0.5
```
Comment (SimonDreher) [inline charts/azure-metrics-exporter/Chart.yaml:6]: As this is a breaking change in the helm chart, I would suggest to bump the major version.

And maybe start a simple changelog file with something like "Defaults for a metricsProbe are now under `probeDefaults`. Move everything under `prometheus.metricProbes` to `prometheus.metricProbes.probeDefaults`, except for `prometheus.metricProbes.probes`"

Apart from that: looks good to me, but I'm just an user :)

**Comment by mblaschke** [comment]
sorry, i was confused about some parts of the initial PR and i don't know why i suggested the `probeDefaults` change.. didn't paid enough attention in the first place while being too busy because of my job :(

i'm going to reverse the stuff and merge your feature, sorry about that

**Pull request merged**